doc/modules/recommended_practices.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+Recommended Practices
+=====================
+
+
+Feature Selection
+=================
+
+**Wrong Practice: Selecting features using Whole Dataset**
+
+    >>> X = numpy.random.randn(500,10000)
+    >>> y = numpy.random.choice(2,500)
+    >>> X_ = SelectKBest(k=25).fit_transform(X,y)
```

Comment:
Better use a more explicit variable name:

```python
X_selected = = SelectKBest(k=25).fit_transform(X, y)
```

Note also the PEP8 style: " " after "," in `fit_transform` call.